Remove `_FORCE_INLINE_` from `TextServer*::_ensure*` methods

aaronp64 opened the pull request:
Reduces binary size by ~1.5 MB when compiling with `TextServerAdvanced`, and ~800 KB for `TextServerFallback`

Did some quick checks profiling `EditorInspector::update_tree()` to see impact when adding a bunch of controls, `TextServerAdvanced` methods using `_ensure_cache_for_size` seem to take around the same percentage of time as before, not seeing any obvious differences.  Let me know if this needs more thorough benchmarking, or if there's any specific path that should be checked.

<!--
Please target the `master` branch in priority.

Relevant fixes are cherry-picked for stable branches as needed by maintainers.

To speed up the contribution process and avoid CI errors, please set up pre-commit hooks locally:
https://docs.godotengine.org/en/latest/contributing/development/code_style_guidelines.html
-->

Ivorforce reviewed (approved):
This was irking me as well every time I looked at `TextServerAdv`. 
As far as I'm aware this `_FORCE_INLINE_` may actually slightly _reduce_ performance because it wipes code caches rather than just jumping to the appropriate spots.
All in all I'm pretty confident this is a 'free' binary size reduction.

Looks like this needs a rebase though!

aaronp64 committed:
Remove _FORCE_INLINE_ from TextServer*::_ensure* methods

Reduces binary size by ~1.5 MB when compiling with TextServerAdvanced, and ~800 KB for TextServerFallback

Ivorforce commented:
Sorry, looks like it needs another rebase, because of the vulkan updates that got fixed for today!

Repiteo commented:
Thanks!
